### PR TITLE
fixes #8210 feat(nimbus):Added link to metric hub documentation for r…

### LINKED ISSUE #8210: Link to metric hub documentation from Nimbus UI
There are now nice auto generated docs for each outcome in metric hub, example:

https://mozilla.github.io/metric-hub/outcomes/firefox_desktop/picture_in_picture/

Let's link to there from the Nimbus UI where ever outcomes are mentioned by ID

### PR DESCRIPTION
**Pull request by Avi-88**
…espective outcomes

Because

- there are now nice auto generated docs for each outcome in metric hub

This commit

- adds links to each outcome, mapping it to its respective documentation in metric hub

[Video demo of the changes made](https://drive.google.com/file/d/1HLsLRKr1CsenhZ-scp13lzUVVTYAE8bV/view)

### CODE REVIEW

**Comment by Avi-88** [comment]
Hello @yashika-khurana ma'am, please review the code and let me know if any further changes are required, thanks

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
nothing major, thank you @Avi-88 if you will notice before the outcomes were separated with `,` now they are separated with `|`, I would suggest to keep `,` :)

**Comment by Avi-88** [comment]
Updated it to use `,` instead of `|`.

image

File: experimenter/experimenter/nimbus-ui/src/components/Summary/TableOverview/index.tsx [hunk 1]
```diff
@@ -14,12 +14,66 @@ type TableOverviewProps = {
...
+            docSlugs[outcome?.application ?? ""]
+          }/${outcome?.slug}`}
+          rel="noreferrer"
+        >
+          {outcome?.friendlyName}
+        </a>
+      ))
+      .reduce((acc, curr) => (
+        <>
+          {acc} , {curr}
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/components/Summary/TableOverview/index.tsx:73]: you can remove extra space here like `          {acc}, {curr}`

**Review by yashikakhurana — CHANGES_REQUESTED** [review]
one minor minor formatting to look better

**Comment by Avi-88** [comment]
> one minor minor formatting to look better

Hello @yashikakhurana  ma'am, I have made the requested changes please let me know if anymore changes are required, thanks

**Comment by yashikakhurana** [comment]
@Avi-88 no worries, the fail is not related to your code, I re-ran the test on ci it should be good

File: experimenter/experimenter/nimbus-ui/src/components/Summary/TableOverview/index.tsx [hunk 2]
```diff
@@ -14,12 +14,66 @@ type TableOverviewProps = {
...
+        <a
+          key={index}
+          target="_blank"
+          data-testid={`primary-outcome-${outcome?.slug}`}
+          href={`https://mozilla.github.io/metric-hub/outcomes/${
+            docSlugs[outcome?.application ?? ""]
+          }/${outcome?.slug}`}
+          rel="noreferrer"
+        >
+          {outcome?.friendlyName}
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/components/Summary/TableOverview/index.tsx:46]: great job on describing href 🎉

**Review by yashikakhurana — APPROVED** [review]
@Avi-88 thank you, and changes are accurate and test cases are on point :)

**Pull request merged**